### PR TITLE
Retrieve the default EE using pytest

### PR DESCRIPTION
**Pull request by cidrblock**
Ongoing work to centralize the default ee.

1. Use a session-scoped autorun fixture to pull the default EE
2. Remove that work from tox 

additional PRs:

1. move other image names into dockerfiles
2. get rid of the constant being set to a function :)
3. Move from quay to ghcr when centralized

related: #1358 

Note: This PR also limits the number of builder builds we perform, and increases the timeout, as this is a common place for CI failures

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -25,7 +29,7 @@
 
 
 @pytest.fixture(scope="session", name="valid_container_engine")
-def fixture_valid_container_image():
+def fixture_valid_container_engine():
```
Comment (cidrblock) [inline tests/conftest.py:32]: this was just a little unrelated cleanup while in the neighborhood

**Review by ssbarnea — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]
